lib/elastomer/client/percolator.rb

```diff
@@ -0,0 +1,72 @@
+module Elastomer
+  class Client
+
+    class Percolator
+
+      # Create a new Percolator for managing a query.
+      #
+      # client - Elastomer::Client used for HTTP requests to the server
+      # index  - The index name
+      # id     - The _id for the query
+      def initialize(client, index, id)
+        @client = client
+        @index = index
+        @id = id
+      end
+
+      attr_reader :client, :index, :id
```

Comment:
`index` should probably be `index_name` to clarify that it's an index name string and not an `Index` object.